### DISCUSSION TITLE
Deploy offline

### DISCUSSION THREAD

**Discussion by MangMax** [coollabsio/coolify#2940] [Q&A]
How can I deploy this project offline?

**Comment by peaklabs-dev** [comment]
@MangMax What do you mean? You need to pull stuff like docker and install it on your server, so you would need internet. You can deploy it on your own server but still some internet connection is needed.

**Reply by Noriller** [reply]
Same question, but I was trying to deploy behind a proxy that I don't really have control to change.

The biggest problem I had was the service generated urls pointing to coolify.

I was thinking along the lines of creating urls like myserverip/service1 myserverip/service2 and so on
Is that the part I would customize traefik?
